ui.apps/src/main/content/jcr_root/apps/hclecomm/components/structure/header/.content.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:Component"
+    jcr:title="Header Container"
+    sling:resourceSuperType="wcm/foundation/components/responsivegrid"
+    componentGroup=".hidden"/>
```

Comment:
@DeepaliAgrwl : Add cq:htmlTag node too and test 
![image](https://user-images.githubusercontent.com/38690824/79568913-57a3dc00-80d4-11ea-8b3a-d1f1d4380cfe.png)

Reply:
added and tested. It is visible as the wrapper element in soruce code.